cumulusci/tasks/datadictionary.py

```diff
@@ -0,0 +1,281 @@
+import csv
+import xml.etree.ElementTree as ET
+
+from collections import defaultdict
+from cumulusci.core.exceptions import CumulusCIException
+from cumulusci.tasks.github.base import BaseGithubTask
+from cumulusci.utils import download_extract_github_from_repo
+from distutils.version import LooseVersion
+from pathlib import PurePosixPath
+
+
+class GenerateDataDictionary(BaseGithubTask):
+    task_docs = """
+    Generate a data dictionary for the project by walking all GitHub releases.
+    The data dictionary is output as two CSV files.
+    One, in `object_path`, includes the Object Name, Object Label, and Version Introduced,
+    with one row per packaged object.
+    The other, in `field_path`, includes Object Name, Field Name, Field Label, Field Type,
+    Picklist Values (if any), Version Introduced.
+    """
+
+    task_options = {
+        "object_path": {
+            "description": "Path to a CSV file to contain an sObject-level data dictionary."
+        },
+        "field_path": {
+            "description": "Path to a CSV file to contain an field-level data dictionary."
+        },
+    }
+
+    def _init_options(self, kwargs):
+        super()._init_options(kwargs)
+
+        if self.options.get("object_path") is None:
+            self.options[
+                "object_path"
+            ] = f"{self.project_config.project__name} sObject Data Dictionary.csv"
+
+        if self.options.get("field_path") is None:
+            self.options[
+                "field_path"
+            ] = f"{self.project_config.project__name} Field Data Dictionary.csv"
+
+        if not self.project_config.project__git__prefix_release:
+            raise CumulusCIException(
+                "Set the `project__git__prefix_release` property in `cumulusci.yml` to use this task"
+            )
```

Comment:
I'd suggest making this a required task option and specify it as `release_prefix: $project_config.project__git__prefix_release` in the task config in cumulusci.yml.

Reasons:
1. The default from cumulusci.yml will be shown when you run `cci task info`
2. It makes it easier to do a one-off run of the task with a different prefix without editing cumulusci.yml